<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputDialog</class>
 <widget class="QDialog" name="OutputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1344</width>
    <height>910</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2160</width>
    <height>1440</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>GUI for Kavach</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/icon.png</normaloff>:/icon/icon.png</iconset>
  </property>
  <widget class="QLabel" name="img_feed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>720</width>
     <height>640</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>720</width>
     <height>640</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>720</width>
     <height>640</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>680</y>
     <width>741</width>
     <height>178</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QLabel" name="date_label">
      <property name="font">
       <font>
        <family>Roboto Condensed Light</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Roboto Condensed Light</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Time :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="time_label">
      <property name="font">
       <font>
        <family>Roboto Condensed Light</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Roboto Condensed Light</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Date :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="number_plate">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>60</y>
     <width>400</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>120</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>670</y>
     <width>451</width>
     <height>187</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="name">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="accuracy">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Accuracy :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mis">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>new :</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="name_label">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string> - </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="acc_label">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string> -</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mis_label">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string> - </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
